requirements.txt

```diff
@@ -0,0 +1,4 @@
+beautifulsoup4
+grr-api-client
+pytz
+requests
```

Comment:
Did you compile this manually or did you generate it using pipenv? (I think `pipenv lock --requirements`)